assets/javascripts/discourse/components/post-voting-comment-composer.hbs

```diff
@@ -0,0 +1,33 @@
+<div class="post-voting-comment-composer">
+  <textarea
+    class="post-voting-comment-composer-textarea"
+    value={{this.value}}
+    {{on "input" this.onInput}}
+    {{on "keydown" @onKeyDown}}
+  ></textarea>
+
+  {{#if this.value.length}}
+    {{#if this.isValueTooShort}}
+      <div class="post-voting-comment-composer-flash error">
+        {{i18n
+          "post_voting.post.post_voting_comment.composer.too_short"
+          count=this.siteSettings.min_post_length
+        }}
+      </div>
+    {{else if this.isValueTooLong}}
+      <div class="post-voting-comment-composer-flash error">
+        {{i18n
+          "post_voting.post.post_voting_comment.composer.too_long"
+          count=this.siteSettings.qa_comment_max_raw_length
+        }}
+      </div>
```

Comment:
Wouldn't it be better to set which error should be displayed in a JS getter and avoid the `else if` here in the template?

Reply:
Yep, sounds good, changed to an `errorMessage` getter.